controller/cache/dlp_rule.go

```diff
@@ -1362,3 +1362,45 @@ func (m CacheMethod) GetDlpRuleNames() *[]string {
 	}
 	return nil
 }
+
+// caller owns cacheMutexRLock & has readAll right
+func (m CacheMethod) GetFedDlpGroupSensorCache() ([]*share.CLUSDlpSensor, []*share.CLUSDlpGroup) {
+	feddlpsensors := make([]*share.CLUSDlpSensor, 0)
+	for _, cdr := range dlpSensors {
+		if cdr.CfgType == share.FederalCfg && strings.HasPrefix(cdr.Name, api.FederalGroupPrefix) {
```

Comment:
Consider making `strings.HasPrefix(xxx, api.FederalGroupPrefix)` a function?

Reply:
this code is not limited in this PR, we can overhaul this practice in our code space and make modification in different pull request,thanks